components/FilterRemedies.js

```diff
@@ -0,0 +1,33 @@
+import symptoms from "../assets/symptoms.json";
+import Link from "next/link";
+
+export default function FilterList({
+  selectedSymptom,
+  handleSymptomChange,
+  handleClearFilter,
+}) {
+  return (
+    <>
+      <section>
+        <select
+          id="symptoms"
+          name="symptoms"
+          value={selectedSymptom}
+          onChange={handleSymptomChange}
+        >
+          <option value="">Filter by Symptom</option>
+          {symptoms.map((symptom, index) => (
+            <option key={index} value={symptom}>
+              {symptom}
+            </option>
+          ))}
+        </select>
+
+        {selectedSymptom && (
+          <button onClick={handleClearFilter}>Remove Filter</button>
```

Comment:
Being explicit about which type of button you are working with is a good practice: 

```suggestion
          <button type="button" onClick={handleClearFilter}>Remove Filter</button>
```